misc/travis-check.sh

```diff
@@ -60,6 +60,9 @@ if [ "$TARGET" = "Unix" ]; then
 		if ! git diff --quiet docs/man; then
```

Comment:
I think we don't need --quiet option here.
If everything works fine, nothing is printed even if we remove --quiet option.
If something wrong, git diff without --quiet gives us much hint for finding the root cause.

Reply:
I will push a change what I think here though I should make another pull request for the purpose.

Reply:
Without --quiet, git doesn't seem to return the error code even the differences are found.

Reply:
Oh, I see.

Reply:
It looks a bug of git for me:-P.

Reply:
```
       --exit-code
           Make the program exit with codes similar to diff(1). That is, it exits with 1 if
           there were differences and 0 means no differences.

       --quiet
           Disable all output of the program. Implies --exit-code.
```

So I think we can use --exit-code.